front/app/i18n/ar-MA.ts

```diff
@@ -0,0 +1,15 @@
+import { addLocaleData } from 'react-intl';
+import ar from 'react-intl/locale-data/ar';
+
+import { formatTranslationMessages } from './';
+
+addLocaleData(ar);
```

Comment:
I found that trying to use `ma` here (again going against our convention of using the first pair of letters, not second pair) resulted in a webpack error:
```
webpack 5.68.0 compiled with 3 errors in 2116 ms
ERROR in app/i18n/ar-MA.ts:2:16
TS2307: Cannot find module 'react-intl/locale-data/ma' or its corresponding type declarations.
    1 | import { addLocaleData } from 'react-intl';
  > 2 | import ma from 'react-intl/locale-data/ma';
      |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    3 |
    4 | import { formatTranslationMessages } from './';
```
... because no such type def exists in `front/node_modules/@types/react-intl/index.d.ts`. I thought about creating the type def there, but guessed that I might be trying to link to locale data from a library that does not exist if I did that.

I eventually settled for modelling the import on the approach in [ar-SA.ts](https://github.com/CitizenLabDotCo/citizenlab/blob/master/front/app/i18n/ar-SA.ts), working on the assumption that this library relates to things like number and date format and thus, `ar` probably works fine?? (Advice / opinion would be gratefully received)

Reply:
I *think* the difference here is locale formatting conventions vs. local differences within a language. Time etc. will be formatted the same in English or (I assume) Arabic regardless of which dialect you're speaking, but the translations for strings might vary based on local differences (e.g. color vs colour). I'd assume that `ar` is fine here as it's the overarching locale, which encompasses the various differences in local dialects within it